lib/css/exports/_stacks-constants-type.less

```diff
@@ -13,9 +13,71 @@
 //  ============================================================================
 //  $   FONT FAMILY (ff-)
 //  ----------------------------------------------------------------------------
+
+// Add Segoe UI Adjusted as a font family, and adjust its baseline
+@font-face {
+    font-family: "Segoe UI Adjusted";
+    src: local(Segoe UI);
+    ascent-override: 95%;
+}
+
+// TODO Segoe UI is variable, but only in Windows 11, so add all the variants
+// TODO add Light/Semilight variants: https://docs.microsoft.com/en-us/typography/font-list/segoe-ui
```

Comment:
Are you interested in adding these variants as well? If not, we can ditch both of these TODO comments